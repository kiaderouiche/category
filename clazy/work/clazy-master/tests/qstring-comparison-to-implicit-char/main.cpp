#include <QtCore/QObject>
#include <QtCore/QString>

void test()
{
    QString s;
    if (s == 5000) {} // Warn
    if (s == 'a') {} // OK
    if (s == "ok") {} // OK
    if (s == s) {} // OK
}
