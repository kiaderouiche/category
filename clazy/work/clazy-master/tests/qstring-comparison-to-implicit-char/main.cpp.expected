qstring-comparison-to-implicit-char/main.cpp:7:9: warning: QString being compared to implicit QChar [-Wclazy-qstring-comparison-to-implicit-char]
